frontend/test/metabase-visual/notebook/notebook.cy.spec.js

```diff
@@ -92,7 +92,7 @@ describe("visual tests > notebook > Run buttons", () => {
     cy.findByText("SQL query").click();
 
     // Check that we're on the blank question page
-    cy.findByText("Here's where your results will appear");
+    cy.findByText("Here's where your results will appear").click();
```

Comment:
Is this copy clickable at all? 🤔

Reply:
> Is this copy clickable at all? 🤔

Nope, I added it to cause blur on the ace editor which did not work through getting the node and triggerting a blur event